src/Libraries/RevitNodes/Elements/Views/View.cs

```diff
@@ -176,6 +177,195 @@ private static string ViewTypeString(ViewType vt)
                     return "Section View";
             }
         }
+        #endregion
+
+        #region Discipline
+
+        private static string ViewDisciplineString(ViewDiscipline vd)
+        {
+            string discipline = "";
+            switch(vd)
+            {
+                case ViewDiscipline.Architectural:
```

Comment:
Could you try to directly transform Enum value to string?

Reply:
OK, I will update soon.